debug/testlib.py

```diff
@@ -1217,7 +1217,7 @@ def postMortem(self):
             return
         self.gdb.interrupt()
         self.gdb.command("info breakpoints", reset_delays=None)
-        self.gdb.command("disassemble", ops=20, reset_delays=None)
+        self.gdb.command("x/20i $pc", ops=20, reset_delays=None)
```

Comment:
Ah, so disassemble doesn't always give useful output if the pc is outside a known source-level function.  This new way looks better, for the situation.

Reply:
And it gives too verbose output if you're in a large function, when it disassembles the whole thing.